<!-- -->
<launch>
    <param name="/baseline" value="0.10" />
    <node pkg="disparity_expansion" exec="disparity_expansion" name="disparity_expansion">
        <param name="lut_max_disparity" value="100" />
        <param name="robot_radius" value="0.25" />
        <param name="bg_multiplier" value="2.0" />
        <param name="padding" value="-1" />
        <param name="sensor_pixel_error" value="0.2" />
	<remap from="/nerian_sp1/right/camera_info" to="/camera/depth/camera_info"/>
	<remap from="/nerian_sp1/disparity_map_32F" to="/camera/disparity_sensor_msg/image_raw"/>	
    </node>
    <node pkg="disparity_expansion" exec="disparity_pcd" name="disparity_pcd_real">
	<remap from="/nerian_sp1/left/image_raw" to="/camera/rgb/image_raw"/>	
	<remap from="/disparity" to="/camera/disparity_sensor_msg/image_raw"/>
	<remap from="/nerian_sp1/right/camera_info" to="/camera/depth/camera_info"/>
	<remap from="/pcd" to="/pcd_real"/>
        <param name="downsample_scale" value="1.0" />
    </node>
    <node pkg="disparity_expansion" exec="disparity_pcd" name="disparity_pcd_fg">
	<remap from="/nerian_sp1/left/image_raw" to="/camera/rgb/image_raw"/>	
	<remap from="/disparity" to="/ceye/left/expanded_disparity_fg"/>
	<remap from="/pcd" to="/pcd_fg"/>
        <param name="downsample_scale" value="2.0" />
	<remap from="/nerian_sp1/right/camera_info" to="/camera/depth/camera_info"/>
    </node>
    <node pkg="disparity_expansion" exec="disparity_pcd" name="disparity_pcd_bg">
	<remap from="/nerian_sp1/left/image_raw" to="/camera/rgb/image_raw"/>	
	<remap from="/disparity" to="/ceye/left/expanded_disparity_bg"/>
	<remap from="/pcd" to="/pcd_bg"/>
        <param name="downsample_scale" value="2.0" />
	<remap from="/nerian_sp1/right/camera_info" to="/camera/depth/camera_info"/>
    </node>
</launch>
